cartridges/int_algolia/cartridge/scripts/algolia/model/algoliaProduct.js

```diff
@@ -257,9 +259,12 @@ var aggregatedValueHandlers = {
         return productPrice;
     },
     in_stock: function (product) {
-        // the in_stock property now exports the SFCC attribute product.availabilityModel.inStock,
-        // not whether the threshold defined in Algolia_InStockThreshold value is reached.
-        return product.availabilityModel.inStock;
+        // the in_stock property now exports a boolean indicating whether the number of products in stock
+        // (ATS or available to sell) is higher than the threshold set in the custom site preference Algolia_InStockThreshold,
+        // as originally intended. The previous logic was flawed in that it compared the sitepref to product.availabilityModel.availability,
+        // which is a number between 0 and 1 and does not represent the number of products in stock.
```

Comment:
This kind of comment is not useful for someone downloading the cartridge for the first time. Especially a comment pointing out previous bugs :thinking:. I don't see the point of explaining what former bugs were doing in code comments, git history and PR descriptions are here for that. As I already suggested on #39, please remove.

```suggestion
```

Reply:
It is critical for someone upgrading though. They should definitely not miss this change as it's a breaking one.

Reply:
That's what the changelog is for.